common/src/main/java/com/wynntils/mc/render/Texture.java

```diff
@@ -16,7 +16,11 @@ public enum Texture {
     LOOTRUN_LINE("path_arrow.png", 16, 16),
     MANA_BAR("bars_mana.png", 256, 256),
     OVERLAY_SELECTION_GUI("overlay_selection_gui.png", 195, 256),
-    WYNNCRAFT_ICON("wynncraft_icon.png", 64, 64);
+    WYNNCRAFT_ICON("wynncraft_icon.png", 64, 64),
+
+    WYNN_MAP_TEXTURES("map/wynn_map_textures.png", 256, 256),
```

Comment:
Could you order this alphabetically? (and perhaps have a comment after newline (`//map textures`?

Reply:
Will do